addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -1,223 +1,256 @@
+        .note-editable, .note-statusbar {
+            border: 1px solid $border-color;
+            border-width: 0 1px;
+        }
 
-        > .bg-success {
-            @include o-position-absolute(0, 0, 0, 0);
+        .note-statusbar {
+            display: block;
+            border-bottom-width: 1px;
         }
-        > span {
-            @include o-position-absolute(10px, 50%);
-            transform: translateX(50%);
-            font-size: 20px;
+    }
 
-            &[data-href] {
-                cursor: pointer;
+    .o_wforum_author_box {
+        &.o_show_info {
+            line-height: 1.2;
+            img {
+                @include size(2em);
             }
+        }
 
-            i {
-                padding: 2px 3px;
-                border-radius: 100%;
+        &.o_compact {
+            line-height: 1;
+
+            img {
+                @include size(1.4em);
             }
         }
-    }
 
-    .user_name_people {
-        font-weight: bold;
-        max-width: 60%;
     }
 
-    .oe_answer_true {
-        color: theme-color('success');
-        &:hover {
-            color: lighten(theme-color('success'), 20%);
+    .forum_answer {
+        .o_wforum_answer_correct_badge {
+            display: none;
         }
-    }
 
-    .oe_answer_false {
-        color: $gray-400;
-        &:hover {
-            color: $gray-600;
+        .o_wforum_author_box_check {
+            @include size(1em);
+            display: none;
+            top: 0;
+            right: -3px;
+            box-shadow: 0 0 0 2px white;
+            line-height: .8;
+        }
+
+        .o_wforum_validate_toggler {
+            @include o-hover-text-color(gray('400'), lighten(theme-color('success'), 20%));
         }
-    }
 
-    .favourite_question {
-        color: $gray-400 !important;
+        &.o_wforum_answer_correct {
+            .o_wforum_answer_correct_badge, .o_wforum_author_box .o_wforum_author_box_check {
+                display: inline;
+            }
+
+            .o_wforum_validate_toggler {
+                @include o-hover-text-color(theme-color('success'), theme-color('warning'));
+
+                &:hover .fa.fa-check:before {
+                    content: '\f00d';
+                }
+            }
 
-        &.forum_favourite_question {
-            color: $gold !important;
+            .o_wforum_answer_header .o_wforum_author_pic {
+                border: 2px solid $success;
+                padding: 3px;
+            }
         }
     }
 
-    .oe_grey {
-        background-color: #eeeeee;
+    .o_wforum_follow_toggler {
+        // Reduce size
+        button, input {
+            padding: $input-btn-padding-y-sm $input-btn-padding-x-sm;
+            font-size: $input-btn-font-size-sm;
+            line-height: $input-btn-line-height-sm;
+        }
+
+        // Hide button and show 'bell' icon for logged-in users that
+        // are not followers.
+        .o_connected_user & > div[data-follow="off"] > div {
```

Comment:
Should find another way

Reply:
Can't xpath `website_mail.follow` without changing its content for all others modules too. What do you suggest?

Reply:
Why not the bell in website_mail directly ?

Reply:
I'm hesitant because this bell design is tailored for this section only.
In the tags page, for example, we should keep the button... the same in website_blog and the event modules.

An option may be to refactor the original  template ad trigger the "icons only" design using a qweb variable.... eg `<t t-if="icon_design">`
Or maybe something like this but I'm not sure if it's suitable
https://github.com/odoo/odoo/blob/12.0/addons/website_mail_channel/views/website_mail_channel_templates.xml#L53

Reply:
@stefanorigano are you working on this? :)

Reply:
@qsm-odoo , not yet. Realistically I may have time tomorrow afternoon. 
If it's urgent I'd suggest to proceed and I'll check the design afterwards.

Reply:
@qsm-odoo , please have a look to this test https://github.com/odoo/odoo/pull/35749/commits/3cef6104284478bcaa7d250d1546da1fe80430ac
thanks

Reply:
Seems ok to me, just don't name the new class `js_follow_icons_container` but something respecting our convention like `o_wmail_follow_icons_container`